src/OpenTelemetry/CHANGELOG.md

```diff
@@ -12,6 +12,11 @@
   function when configuring a view (applies to individual metrics).
   ([#5542](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5542))
 
+* **Experimental (pre-release builds only):** The default `ExemplarFilterType`
```

Comment:
Could we reword this to say something like "Exemplars will be enabled automatically with default filter type set to TraceBased and can be changed by calling SetExemplarFilter on MeterProvider".

Reply:
I tweaked it a bit